multisig/proposals/arbitrumsepolia/vip-004/index.ts

```diff
@@ -0,0 +1,79 @@
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { arbitrumsepolia } = NETWORK_ADDRESSES;
+
+const XVS_STORE = "0x4e909DA6693215dC630104715c035B159dDb67Dd";
+export const ACM = "0xa36AD96441cB931D8dFEAAaC97D3FaB4B39E590F";
+
+const vip004 = () => {
+  return makeProposal([
+    {
+      target: arbitrumsepolia.XVS_VAULT_PROXY,
+      signature: "_acceptAdmin()",
+      params: [],
+    },
+
+    {
+      target: XVS_STORE,
+      signature: "acceptAdmin()",
+      params: [],
+    },
+
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.XVS_VAULT_PROXY, "pause()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.XVS_VAULT_PROXY, "resume()", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [
+        arbitrumsepolia.XVS_VAULT_PROXY,
+        "add(address,uint256,address,uint256,uint256)",
+        arbitrumsepolia.NORMAL_TIMELOCK,
+      ],
+    },
+
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [arbitrumsepolia.XVS_VAULT_PROXY, "set(address,uint256,uint256)", arbitrumsepolia.NORMAL_TIMELOCK],
+    },
+
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [
+        arbitrumsepolia.XVS_VAULT_PROXY,
+        "setRewardAmountPerBlockOrSecond(address,uint256)",
+        arbitrumsepolia.NORMAL_TIMELOCK,
+      ], // func name changed from setRewardAmountPerBlock to setRewardAmountPerBlockOrSecond
+    },
+
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [
+        arbitrumsepolia.XVS_VAULT_PROXY,
+        "setWithdrawalLockingPeriod(address,uint256,uint256)",
+        arbitrumsepolia.NORMAL_TIMELOCK,
+      ],
+    },
+
+    {
+      target: arbitrumsepolia.XVS_VAULT_PROXY,
+      signature: "add(address,uint256,address,uint256,uint256)",
+      params: [arbitrumsepolia.XVS, 100, arbitrumsepolia.XVS, "0", 604800],
+    },
```

Comment:
I would add the command to pause the vault. On mainnet, the vault will be paused for some days, to allow users to bridge XVS from other networks. Some days later, Prime will be enabled and the vault will be enabled, and users will be able to stake their XVS, starting the qualification period for Prime at the same time.

That was the process followed for Ethereum mainnet. By pausing the XVSVault initially, we'll be able to test the UI with this paused status

Reply:
See [this](https://github.com/VenusProtocol/vips/blob/develop/multisig/proposals/ethereum/vip-004/index.ts#L67) as a reference

Reply:
Got it. Added the command: https://github.com/VenusProtocol/vips/pull/250/commits/e9896a8fa19d150d8444164ea4a866ed95e8c8b5